docker/README.md

```diff
@@ -0,0 +1,28 @@
+# OSS Review Toolkit Docker Files
+
+This directory contains several Docker files that ease building and / or running ORT without the need to install any
+prerequisites except for Docker. See below for descriptions of the different files and their purposes.
+
+## build/Dockerfile
+
+The Docker file in the `build` directory is for *building* ORT locally on a machine with direct Internet access, i.e. no
+proxy setup is required. It is not meant to be used directly, but via the `build.sh` script, which builds the image and
+runs it *as the current user* with the user's `$HOME` directory mounted into the container. It writes build output back
+to the host just as if you had built ORT locally as the current user.
+
+Example usage:
+
+    $ docker/build.sh
+
+## run/Dockerfile
+
+The Docker file in the `run` directory is for *running* ORT locally on a machine with direct Internet access, i.e. no
+proxy setup is required. It is not meant to be used directly, but via the `run.sh` script, which first builds ORT via
+the `build.sh` script (if not already done so), and then runs ORT with the first (quoted) argument being passed to
+`docker run` and all remaining arguments being passed to ORT.
+
+Example usages:
+
+    $ docker/run.sh "-v $WORKSPACE/project:/project" analyze -f JSON -i /project/source -o /project/ort/analyzer
```

Comment:
Can you add a sentence about mounting the project directory? I think that might be a point where users might be confused about what they need to mount and where. And maybe remove the `$WORKSPACE` variable, it is unclear where it is coming from and not required here.

Reply:
Done.